lib/cacheable/response_cache_handler.rb

```diff
@@ -3,18 +3,28 @@
 
 module Cacheable
   class ResponseCacheHandler
-    def initialize(**kwargs, &block)
+    def initialize(
+      key_data:,
+      version_data:,
+      env:,
+      cache_age_tolerance:,
+      serve_unversioned:,
+      headers:,
+      force_refill_cache: false,
+      cache_store: Cacheable.cache_store,
```

Comment:
Not sure if this is a concern but `kwargs[:cache_store] || Cacheable.cache_store` isn't the same as this.

Reply:
Right. But all users of this class that I could find don't even pass this argument, so in practice it is the same.